tests/license_check.py

```diff
@@ -9,6 +9,12 @@
     "tiktoken",
     # openai as Apache 2.0; for some reason, after updating to 1.0, pip-licenses thinks it's UNKNOWN
     "openai",
+    # UNKNOWN
+    "chroma-hnswlib",
+    "pyinstaller-hooks-contrib",
+    # "Other/Proprietary License"
+    "pinecone-client",  # part of chromadb - we don't use directly, but potentially problematic
+    "pygls",
```

Comment:
Kindof a mess. Thoughts?

Reply:
As long as we're compatible with all of their licenses it's fine; have you checked the licenses on their github? We mainly just need to make sure none of them have copyleft licenses like GPL